test/plugin/test_in_exec.rb

```diff
@@ -130,4 +146,15 @@ def test_emit_regexp
     assert_equal ["regex_tag", @test_time, {"message"=>"hello"}], emits[0]
     assert_equal @test_time, emits[0][1]
   end
+
+  def test_emit_with_invalid_script
+    d = create_driver invalid_json_config
+
+    d.run do
+      sleep 2
+    end
+
+    assert_equal true, d.emits.empty?
+    assert_equal true, File.read(@count_file).length.between?(1, 4)
```

Comment:
For counting command execution, checking log lines seems enough like below:

https://github.com/fluent/fluentd/blob/e307297d88dc2890af28319faab30baead304988/test/plugin/test_in_forward.rb#L167